File: boot-tyb/src/main/java/com/xzm/tyb/pojo/vo/PageVo.java
package com.xzm.tyb.pojo.vo;
import io.swagger.annotations.ApiModelProperty;

public class PageVo extends AuthVo {
    @ApiModelProperty(value = "分页pageNum")
    private   int pageNum = 1;
    @ApiModelProperty("分页pageSize")
    private int pageSize = 5;

    public int getPageNum() {
        return pageNum;
    }

    public void setPageNum(int pageNum) {
        this.pageNum = pageNum;
    }

    public int getPageSize() {
        return pageSize;
    }

    public void setPageSize(int pageSize) {
        this.pageSize = pageSize;
    }
}

File: boot-tyb/src/main/java/com/xzm/tyb/pojo/vo/PageVo.kt
package com.xzm.tyb.pojo.vo

import io.swagger.annotations.ApiModelProperty

open class PageVo : AuthVo() {
    @ApiModelProperty(value = "分页pageNum")
    var pageNum = 1
    @ApiModelProperty("分页pageSize")
    var pageSize = 10
}
